src/include/optimizer/statistics/value_condition.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "catalog/catalog_defs.h"
+#include "loggers/optimizer_logger.h"
+#include "parser/expression_defs.h"
+#include "type/transient_value.h"
+
+namespace terrier::optimizer {
+/**
+ * ValueCondition
+ * This class stores the expression type and value used in the WHERE clause as well as the
+ * column name/id it affects.
+ *
+ * SELECT * FROM table WHERE [id = 1] <- ValueCondition
+ */
+class ValueCondition {
+ public:
+  /**
+   * Constructor
+   * @param column_id
```

Comment:
You need descriptions of the parameters. Don't assume that it is obvious based on the name.